blocks/header/header.css

```diff
@@ -594,6 +527,177 @@ header .nav-bottom .other-items .mobile-lang-picker .mobile-lang-picker-item a {
   letter-spacing: -0.4px;
 }
 
+/* stylelint-disable-next-line no-descending-specificity */
+header .drop > a::after {
```

Comment:
There is one more style 
**header nav.nav-bottom .menu-level-1-item:first-of-type > a::after** may be u can remove this